Packs/AzureStorageContainer/Integrations/AzureStorageContainer/AzureStorageContainer.py

```diff
@@ -0,0 +1,921 @@
+
+    for element in root.iter('Tag'):
+        tag = {'Key': element.findtext('Key'), 'Value': element.findtext('Value')}
+        raw_response.append(dict(tag))
+
+    outputs['Blob']['Tag'] = raw_response
+
+    readable_output = tableToMarkdown(
+        f'Blob {blob_name} Tags:',
+        outputs['Blob']['Tag'],
+        headers=['Key', 'Value'],
+        headerTransform=pascalToSpace
+    )
+
+    command_results = CommandResults(
+        readable_output=readable_output,
+        outputs_prefix='AzureStorageContainer.Container',
+        outputs_key_field='name',
+        outputs=outputs,
+        raw_response=raw_response
+    )
+
+    return command_results
+
+
+def create_set_tags_request_body(tags: dict) -> str:
+    """
+    Create XML request body for set blob tags.
+    Args:
+        tags (dict): Tags data. Key represents tag name , and value represents tag Value.
+
+    Returns:
+        str: Set tags request body.
+
+    """
+    top = ET.Element('Tags')
+
+    tag_set = ET.SubElement(top, 'TagSet')
+
+    for key, value in tags.items():
+        tag = ET.SubElement(tag_set, 'Tag')
+        tag_key = ET.SubElement(tag, 'Key')
+        tag_key.text = key
+
+        tag_value = ET.SubElement(tag, 'Value')
+        tag_value.text = value
+
+    return ET.tostring(top, encoding='unicode')
+
+
+def set_blob_tags_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+    Sets the tags for the specified Blob.
+
+    Args:
+        client (Client): Azure Blob Storage API client.
+        args (dict): Command arguments from XSOAR.
+
+    Returns:
+        CommandResults: outputs, readable outputs and raw response for XSOAR.
+
+    """
+    container_name = args['container_name']
+    blob_name = args['blob_name']
+    tags = args['tags']
+
+    try:
+        tags = json.loads(tags)
+    except ValueError:
+        raise ValueError('Failed to parse tags argument. Please provide valid JSON format tags data.')
+
+    xml_data = create_set_tags_request_body(tags)
+
+    client.set_blob_tags_request(container_name, blob_name, xml_data)
+
+    command_results = CommandResults(
+        readable_output=f'{blob_name} Tags successfully updated.',
+    )
+
+    return command_results
+
+
+def delete_blob_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+    Delete Blob from Container.
+
+    Args:
+        client (Client): Azure Blob Storage API client.
+        args (dict): Command arguments from XSOAR.
+
+    Returns:
+        CommandResults: outputs, readable outputs and raw response for XSOAR.
+
+    """
+    container_name = args['container_name']
+    blob_name = args['blob_name']
+
+    client.delete_blob_request(container_name, blob_name)
+
+    command_results = CommandResults(
+        readable_output=f'Blob {blob_name} successfully deleted.',
+    )
+
+    return command_results
+
+
+def replace_dict_keys(data: dict, keys: list) -> dict:
```

Comment:
This function is meant to transform data to context suitable standards, right?
let's name it appropriately to describe that

Reply:
Done.